lib/utils.js

```diff
@@ -56,8 +56,10 @@ const buildUrl = (host, port, ssl, endpoint) => {
 };
 
 // Custom Logger
-const log = (msg) => {
-    console.log('[' + new Date().toISOString().substring(2, 19).replace('T', ' ').replace(/[Z-]/g, '') + '] ' + msg);
+const log = (message, level) => {
+    let datetime = new Date();
+    datetime = `${datetime.toISOString().replace('T', ' ').split('.')[0]} ${datetime.toString().split(' ')[5].split('GMT')[1]}`;
```

Comment:
This is not being built correctly. You're using the UTC parts to fill in the `YYYY-MM-DD HH:MM:SS` portion, but then using the local timezone offset which results in an invalid date.

Also this seems like a roundabout way of building the string, why not just access the individual portions e.g. `datetime.getDate()`, `datetime.getMonth()`, etc...

Reply:
@jakedipity, updated! It's now correctly matching what `(new Date()).toString()` is showing in terms of local time and timezone